### PR TITLE
IBX-8205: Added possibility to configure identifier of dashboard container

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://github.com/ibexa/dashboard/pull/126)
| Versions      | master, 4.6

Added possibility to configure identifier of dashboard container.

Preview: https://ez-systems-developer-documentation--2404.com.readthedocs.build/en/2404/administration/dashboard/configure_default_dashboard/

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Code samples are working

### CODE REVIEW

**Review by mnocon — COMMENTED** [review]
Not every Content Type can be configured as a dashboard container. 

Can we mention that the custom Content Type needs to have a field type with `name` identifier?

**Comment by mikadamczyk** [comment]
> Not every Content Type can be configured as a dashboard container.
> 
> Can we mention that the custom Content Type needs to have a field type with `name` identifier?

It just came to my mind that it should be a container

**Review by mnocon — APPROVED** [review]
Thanks!

**Review by mikadamczyk — APPROVED** [review]

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 1]
```diff
@@ -70,4 +70,16 @@ You can find it in the Admin panel, **Dashboard content Ttpe** section, **View/G
...
+
+```yaml
+ibexa:
+    system:
+        default:
+            dashboard:
+                container_content_type_identifier: user_dashboard_container
+```
+
+Custom content type should be a container and needs to have a field type with `name` identifier.
```
Comment (dabrt) [inline docs/administration/dashboard/configure_default_dashboard.md:85]:
```suggestion
The custom content type should be a container and needs to have a field type with `name` identifier.
```

**Review by dabrt — APPROVED** [review]

**Pull request merged**